website/components/CommunityCompatibleTable.tsx

```diff
@@ -0,0 +1,160 @@
+      status: CompatibleStatus.Compatible,
+    },
+    {
+      name: 'eslint-webpack-plugin',
+      url: 'https://github.com/webpack-contrib/eslint-webpack-plugin',
+      status: CompatibleStatus.Compatible,
+    },
+    {
+      name: 'webpack-manifest-plugin',
+      url: 'https://github.com/shellscape/webpack-manifest-plugin',
+      status: CompatibleStatus.NotCompatible,
+      description: i18n[lang]['webpack-manifest-plugin-desc'],
+    },
+    {
+      name: 'copy-webpack-plugin',
+      url: 'https://www.npmjs.com/package/copy-webpack-plugin',
+      status: CompatibleStatus.Included,
+      description: i18n[lang]['copy-plugin-desc'],
+    },
+    {
+      name: 'mini-css-extract-plugin',
+      url: 'https://webpack.js.org/plugins/mini-css-extract-plugin',
+      status: CompatibleStatus.Included,
+      description: i18n[lang]['mini-css-extract-plugin-desc'],
+    },
+    {
+      name: 'terser-webpack-plugin',
+      url: 'https://webpack.js.org/plugins/terser-webpack-plugin',
+      status: CompatibleStatus.NotCompatible,
+      description: i18n[lang]['terser-webpack-plugin-desc'],
+    },
+    {
+      name: 'webpack-bundle-analyzer',
+      url: 'https://www.npmjs.com/package/webpack-bundle-analyzer',
+      status: CompatibleStatus.Compatible,
+    },
+    {
+      name: 'tsconfig-paths-webpack-plugin',
+      url: 'https://www.npmjs.com/package/tsconfig-paths-webpack-plugin',
+      status: CompatibleStatus.Included,
+      description: i18n[lang]['tsconfig-paths-webpack-plugin-desc'],
+    },
+    {
+      name: 'fork-ts-checker-webpack-plugin',
+      url: 'https://github.com/TypeStrong/fork-ts-checker-webpack-plugin',
+      status: CompatibleStatus.Compatible,
+    },
+    {
+      name: 'webpack-subresource-integrity',
+      url: 'https://github.com/waysact/webpack-subresource-integrity',
+      status: CompatibleStatus.NotCompatible,
+    },
+  ];
+
+  return (
+    <Table
+      className={S.PluginSupportStatusTable}
+      header={[
+        {
+          name: lang === 'zh' ? '插件' : 'Plugin',
+          key: 'name',
+        },
+        {
+          name: lang === 'zh' ? '支持情况' : 'Support status',
+          key: 'status',
+          style: {
+            width: '200px',
+          },
+        },
+        {
+          name: lang === 'zh' ? '备注' : 'Notes',
+          key: 'notes',
+        },
+      ]}
+      body={pluginList
+        .sort((a, b) => b.status - a.status)
+        .map(({ name, url, status, description }) => {
+          const { symbol, en, zh } = SUPPORT_STATUS_LOCALIZED[status];
+          const statusText = `${symbol} ${lang === 'zh' ? zh : en}`;
+
+          const notesText = (() => {
+            if (description) {
+              return description;
+            }
+            if (status === CompatibleStatus.NotCompatible) {
+              return lang === 'zh' ? '待支持' : 'To be implemented';
+            }
+          })();
+
+          return {
+            name: <a href={url}>{name}</a>,
```

Comment:
Can we highlight the links like this? So the user can know that this is a link

![image](https://github.com/web-infra-dev/rspack/assets/7237365/09b302f8-e0a6-4ced-9c03-2a0e2efd4ad1)

Reply:
updated.

<img width="1024" alt="image" src="https://github.com/web-infra-dev/rspack/assets/22373761/afd13d5c-8bc5-467d-b50f-3a4da16a6840">